src/components/app-configure/general/AppBranding.vue

```diff
@@ -115,16 +115,16 @@ async function handleFileRemove(
       <template #title>Branding</template>
       <VStack direction="column" gap="2rem">
         <VStack direction="column" gap="0.5rem">
-          <h3>Upload Logo</h3>
+          <h3>Wallet Logo</h3>
           <VStack direction="column" gap="0.5rem" class="body-2">
             <p class="body-2 description">
-              The actual shape of the logo does not need to be a rectangle, but
-              the live area should be like a wide rectangle to be used on your
-              website/ app etc.
+              Use a wide rectangular live area for the best viewing results of
```

Comment:
“for the best viewing results” → “to best view the”

Reply:
Done here https://github.com/arcana-network/developer-dashboard/pull/216/commits/8252aad4eb1ac97179e50c6a0526221e36ce3f4f